examples/standalone-in-docker/main.tf

```diff
@@ -18,83 +18,88 @@ resource "docker_container" "mdb0-0" {
-  special     = true
-  min_special = 1
-}
-
 
 # Deploy a single instance of Ops Manager
+locals {
+  ops_manager_port = 8080
+}
 resource "mongodb_opsmanager" "opsman" {
   host {
-    user     = "root"
+    user = "root"
     hostname = "127.0.0.1"
-    port     = docker_container.mdb0-0.ports[0].external
+    port = docker_container.mdb0-0.ports[0].external
   }
 
   opsmanager {
     # https://downloads.mongodb.com/on-prem-mms/deb/mongodb-mms_4.1.8.55362.20190620T1446Z-1_x86_64.deb
-    binary              = "http://localhost:9000/mongodb-mms_4.1.8.55966.20190620T2143Z-1_x86_64.deb"
-    workdir             = "/opt/mongodb"
-    mongo_uri           = "mongodb://${mongodb_process.mdb_standalone.host.0.hostname}:${mongodb_process.mdb_standalone.mongod.0.port}/"
-    encryption_key      = random_string.encryptionkey.result
-    port                = 8080
-    central_url         = "http://${mongodb_process.mdb_standalone.host.0.hostname}:8080"
-    register_first_user = true
-    first_user_password = random_string.globalownerpassword.result
-    ops_manager_port    = docker_container.mdb0-0.ports[1].external
+    binary = "http://localhost:9000/mongodb-mms_4.1.8.55966.20190620T2143Z-1_x86_64.deb"
+    workdir = "/opt/mongodb"
+    mongo_uri = "mongodb://${mongodb_process.mdb_standalone.host.0.hostname}:${mongodb_process.mdb_standalone.mongod.0.port}/"
+    encryption_key = random_string.encryptionkey.result
+    port = local.ops_manager_port
+    external_port = docker_container.mdb0-0.ports[1].external
+    central_url = "http://${mongodb_process.mdb_standalone.host.0.hostname}:${local.ops_manager_port}"
+    register_global_owner = true
+    global_owner_username = "admin"
+    global_owner_password = random_string.globalownerpassword.result
 
     overrides = {
-      "mms.ignoreInitialUiSetup"      = "true"
-      "mms.fromEmailAddr"             = "noreply@example.com"
-      "mms.replyToEmailAddr"          = "noreply@example.com"
-      "mms.adminEmailAddr"            = "noreply@example.com"
-      "mms.mail.transport"            = "smtp"
-      "mms.mail.hostname"             = "localhost"
-      "mms.mail.port"                 = "25"
-      "mms.mail.ssl"                  = "false"
+      "mms.ignoreInitialUiSetup" = "true"
+      "mms.fromEmailAddr" = "noreply@example.com"
+      "mms.replyToEmailAddr" = "noreply@example.com"
+      "mms.adminEmailAddr" = "noreply@example.com"
+      "mms.mail.transport" = "smtp"
+      "mms.mail.hostname" = "localhost"
+      "mms.mail.port" = "25"
+      "mms.mail.ssl" = "false"
       "automation.versions.directory" = "/data/automation/mongodb-releases"
-      "automation.versions.source"    = "mongodb"
+      "automation.versions.source" = "mongodb"
     }
   }
 }
 
+
+# Generate an encryption key for Ops Manager
+resource "random_string" "encryptionkey" {
+  length = 24
+  special = true
+}
+
+resource "random_string" "globalownerpassword" {
+  length = 12
+  min_numeric = 4
+  min_special = 1
```

Comment:
we need to have at least one letter as well, although unlikely we should remove the possibility by adding `min_lower = 1` or `min_upper = 1`

Reply:
good point, unlikely but not impossible :)